github/Installation.py

```diff
@@ -52,7 +53,19 @@ def __repr__(self):
 
     @property
     def id(self):
-        return self._id
+        return self._id.value
+
+    @property
```

Comment:
Would you mind adding docstrings about their types?

Reply:
Why not type hints?  https://docs.python.org/3/library/typing.html

Reply:
We actually do both

Reply:
Why duplicate?  They could easily get out of sync with each other.  Just do one and use mypy to enforce it.

Reply:
Because docstrings are used for our documentation, which I and others refer to first before digging to the .pyi files.